MonoGame.Framework/Platform/Input/Joystick.Web.cs

```diff
@@ -37,6 +37,7 @@ private static JoystickCapabilities PlatformGetCapabilities(int index)
             return new JoystickCapabilities()
             {
                 IsConnected = connected,
+                DisplayName = "",
```

Comment:
Use `string.Empty` here.

Reply:
Fun note, `string.Empty` was only useful for .NET 2 or so, its long since been fixed to be the same as using "", so right now its only about which one you prefer to use.